etc/kayobe/environments/ci-multinode/networks.yml

```diff
@@ -131,6 +131,13 @@ provision_oc_allocation_pool_end: 192.168.33.254
 provision_oc_vlan: 107
 provision_oc_zone: "provision_oc"
 
+octavia_cidr: 192.168.34.0/24
+octavia_vlan: 999
+octavia_allocation_pool_start: 192.168.34.3
+octavia_allocation_pool_end: 192.168.34.9
+octaiva_mtu: "{{ ansible_facts.default_ipv4.mtu - 50 }}"
```

Comment:
```suggestion
octavia_mtu: "{{ ansible_facts.default_ipv4.mtu - 50 }}"
```

Reply:
Good catch. Not sure what the MTU was of the interface though it was working according to the Tempest tests I ran.